wagtail/core/models.py

```diff
@@ -1063,14 +1063,27 @@ def move(self, target, pos=None):
         # Log
         logger.info("Page moved: \"%s\" id=%d path=%s", self.title, self.id, new_url_path)
 
-    def copy(self, recursive=False, to=None, update_attrs=None, copy_revisions=True, keep_live=True, user=None, process_child_object=None, exclude_fields=None):
-        # Fill dict with self.specific values
-        specific_self = self.specific
-        default_exclude_fields = ['id', 'path', 'depth', 'numchild', 'url_path', 'path', 'index_entries']
-        exclude_fields = default_exclude_fields + specific_self.exclude_fields_in_copy + (exclude_fields or [])
-        specific_dict = {}
+    def sync_fields_with(self, other, exclude_fields=None):
```

Comment:
I think for more general usage, it might be nicer to refactor this so it takes a list of fields to sync (so you can sync a single field between pages, for example), rather than exclude - and make the `copy` method do the exclusion part. What do you think?